usr/lib/qubes-whonix/replace-ips

```diff
@@ -208,13 +227,29 @@ def main(args):
                 print("ERROR: 'qubesdb-read /qubes-ip' or 'qubesdb-read /qubes-gateway' failed!")
                 return
 
+            try:
+                ## FIXME: 'qubesdb-read /qubes-ip6' could fail if NetVM is set to 'none'.
+                current_ip6_local = subprocess.check_output(['qubesdb-read', '/qubes-ip6']).decode().rstrip()
+                ## FIXME: 'qubesdb-read /qubes-gateway6' could fail if NetVM is set to 'none'.
+                current_ip6_gateway = subprocess.check_output(['qubesdb-read', '/qubes-gateway6']).decode().rstrip()
+            except (OSError, subprocess.CalledProcessError):
+                print("ERROR: 'qubesdb-read /qubes-ip6' or 'qubesdb-read /qubes-gateway6' failed!")
+                return
+
             if IP_ADDRESS.match(current_ip_local) and IP_ADDRESS.match(current_ip_gateway):
                 ips_to_replace = [last_ip_local, '10.152.152.11']
                 replace_ip(ips_to_replace, current_ip_local, FILES, WHONIX_IP_LOCAL)
 
                 ips_to_replace = [last_ip_gateway, '10.152.152.10']
                 replace_ip(ips_to_replace, current_ip_gateway, FILES, WHONIX_IP_GATEWAY)
 
+            if IP6_ADDRESS.match(current_ip6_local) and IP6_ADDRESS.match(current_ip6_gateway):
+                ips_to_replace = [last_ip_local, 'fd19:c33d:88bc::11']
```

Comment:
Shouldn't `last_ip_local` be `last_ip6_local` here?

Reply:
Correct, fixed